### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by SriPallaviDamuluri**

### CODE REVIEW

File: docs/Groups/member-remove [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# About Member Remove Event
```
Comment (chandur626) [inline docs/Groups/member-remove]: Make sure that the file extension exists as md.

**Comment by nirajlavani** [comment]
doc pages look good to me

**Comment by chandur626** [comment]
relevant doc pages seem good to me.

**Review by chandur626 — APPROVED** [review]

**Review by nirajlavani — APPROVED** [review]
👍👍👍

**Review by HariniBharata — APPROVED** [review]
Approved

**Review by 05sandesh — APPROVED** [review]
code looks good

**Pull request merged**